### PR TITLE
Fix workspace workflows

### PR DESCRIPTION
**Pull request by buchi**
- Allow to create news document versions.
- Allow to trash/untrash documents.

Jira: https://4teamwork.atlassian.net/browse/CA-1070

## Checklist (Must have)

_Everything has to be done/checked. Checked but not present means the author deemed it unnecessary._

- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

### CODE REVIEW

File: opengever/core/upgrades/20200924155829_fix_workspace_workflows/upgrade.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            'portal_type': types,
+            'path': {
+                'query': [root.getPath() for root in workspace_roots],
+                'depth': -1,
+            },
+        }
+        brains = tuple(catalog.unrestrictedSearchResults(query))
+        generator = SizedGenerator(
+            (brain._unrestrictedGetObject() for brain in brains), len(brains))
+        return ProgressLogger('Update object security', generator)
```
Comment (njohner) [inline opengever/core/upgrades/20200924155829_fix_workspace_workflows/upgrade.py:54]: My linter would want a new line here 😅

**Review by njohner — APPROVED** [review]
apart from comment, LGTM. Thanks for the fix!

File: opengever/core/upgrades/20200924155829_fix_workspace_workflows/upgrade.py (hunk 1 above)
Comment (buchi) [reply opengever/core/upgrades/20200924155829_fix_workspace_workflows/upgrade.py:54]: Fixed linting errors.

**Review by njohner — APPROVED** [review]
LGTM, thanks for fixing the other upgrade step.

**Pull request merged**